invenio_cli/cli/packages.py

```diff
@@ -67,11 +67,26 @@ def install(cli_config, packages, skip_build):
 @pass_cli_config
 def outdated(cli_config):
     """Show outdated Python dependencies."""
-    raise click.UsageError("Not supported yet.")
+    steps = PackagesCommands.outdated_packages()
 
+    on_fail = ""
+    on_success = ""
+
+    run_steps(steps, on_fail, on_success)
 
 @packages.command()
+@click.argument("package", nargs=-1, type=str)
 @pass_cli_config
-def update(cli_config):
+def update(cli_config, package):
     """Update a single Python python package."""
-    raise click.UsageError("Not supported yet.")
+
+    if len(package) == 1:
```

Comment:
moderate: i'm not sure the logic is correct... Firstly, does `pipenv` accept more than one package to update? i.e. `pipenv update package1 package2`?

If yes:
- It should do some processing to pass them all (e.g. `" ".join(packages)`)
If no:
- It should check that maximum of one package name is passed

Right now, what would happen is that if the user passes several packages, e.g.
`invenio-cli update invenio-app-rdm invenio-rdm-records`, it will update **all** and it might not be desirable.